tests/Microsoft.DotNet.Framework.Docker.Tests/DockerHelper.cs

```diff
@@ -18,7 +18,7 @@ public DockerHelper(ITestOutputHelper outputHelper)
             OutputHelper = outputHelper;
         }
 
-        public void Build(string tag, string dockerfile, string buildContextPath, StringCollection buildArgs)
+        public void Build(string tag, string dockerfile, string buildContextPath, List<string> buildArgs)
```

Comment:
Why the need to take a dependency on List  This method just needs to enumerate?  IEnumerable{string} allows the caller to pass in a variety of types - arrays, lists, collection, etc..

Reply:
There was not an easy way to add to IEnumerable <string>, that is why I used List here.

Reply:
What is wrong with 

```
public void Build(string tag, string dockerfile, string buildContextPath, IEnumerable<string> buildArgs)
```

Reply:
There is nothing wrong with that.  What I need here is to initialize this collection with one string, and in some situation, Add another string to it.  I did not see that IEnumerable supports Add method.

Reply:
`List<string>` implements `IEnumerable<string>` so you will be able to pass in a `List<string>` type into a method that has an `IEnumerable<string>` parameter.